src/static/js/filter-teams.js

```diff
@@ -0,0 +1,61 @@
+const buttons = document.querySelectorAll('button');
```

Comment:
I think this logic could be simplified. What do you think of an approach like this:

- `div.contributors` is initialized with a class for each team name
```html
<div class="contributors analysts authors brainstormers designers developers editors reviewers translators">
```
- in the CSS we have rules for each team so that if `.contributors.<team-id>` is set, then `.contributor.<team-id>` is set to `display: block` (`.contributor` could be initialized to `display: none`)
```css
.contributor {
  display: none;
}
/* For each team... */
.contributors.developers .contributor.developers {
  display: block;
}
```
- the toggle state of each filter button is linked to the classes on `.contributors`, so every button click results in a `toggleClass(<team-id>)` call
```js
const buttons = document.querySelector('.filter-container');
const contributors = document.querySelector('.contributors')
buttons.addEventListener('click', e => {
  const teamId = e.target.className;
  contributors.classList.toggle(teamId);
});
```
- `div.filter-container` and the toggle buttons could be moved inside of `div.contributors` so they can inherit their styles depending on the selected teams
```css
.contributors.developers button.developers {
  background: #0053ba;
  color: white;
}

Reply:
> /* For each team... */
.contributors.developers .contributor.developers {
  display: block;
}

This might mean having to manually add//edit/delete lines of different teams should there be any changes in the future. 

However, I don't think there will be lots of changes to the current teams' (perhaps none of at all) from time to time. So I agree this approach is much better to read and it also simplifies the logic which is a good thing 😄

Reply:
We could also generate the CSS in an inline style block in the template, which should be more resilient.